twopaneview/src/TwoPaneView.tsx

```diff
@@ -0,0 +1,221 @@
+import React, {Fragment, Component} from 'react';
+import {
+  View,
+  Dimensions,
+} from 'react-native';
+import { DualScreenInfo } from 'react-native-dualscreeninfo'
+import {PaneOrientationTypes, PaneModeTypes, PanePriorityTypes} from "./types"
+
+const TPV_Orientation: PaneOrientationTypes = {
+  Horizontal: 'horizontal',
+  Vertical: 'vertical',
+};
```

Comment:
I'm not an expert on typescript, but it seems like this definition is unnecessary.  I think what we want is to define this as an enum in the .tx file, and then use the type directly from within TwoPaneView code.  This should allow us to remove TPV_Orientaiton altogether.

Same comment applies toTPV_PanePriority and PaneMode.

Reply:
That actually makes so much more sense. Thanks for pointing that out! Just pushed the change.